src/components/navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Input from "../input/Input.component";
+import Menu from "../menu/Menu.component";
+import { StyledNavbarContainer, StyledWrapperNavbar } from "./Navbar.styles";
+import { useGlobalContext } from "../../providers/Global/Global.provider";
+
+// eslint-disable-next-line react/prop-types
+function Navbar() {
+  const { search } = useGlobalContext();
+  const [inputValue, setInputValue] = useState("");
+
+  const handleChange = (event) => {
+    setInputValue(event.currentTarget.value);
+    search(event.currentTarget.value);
+  };
+
+  return (
+    <StyledNavbarContainer title="navbar">
+      <Menu />
+      <StyledWrapperNavbar>
+        <Input
```

Comment:
We could try to turn the search input into a component component, that way we can  reuse it in different parts of a project. Or even in case we want to change its position or the information where we are looking